packages/emptystate/src/react/index.tsx

```diff
@@ -6,13 +6,13 @@ import {
   RefForwardingComponent,
   RefFor,
   ValueOf,
+  classNames,
   useResizeObserver
 } from '@pluralsight/ps-design-system-util'
-import glamorDefault, * as glamorExports from 'glamor'
 import React from 'react'
 
-import Context, { ContextValue } from './context'
```

Comment:
Why remove context?

Reply:
It's not removed. It was reordered. See line 15. ContextValue was removed because it's now unused in this file.